js/src/util/sanitizer.js

```diff
@@ -92,8 +92,8 @@ export const DefaultWhitelist = {
 }
 
 export function sanitizeHtml(unsafeHtml, whiteList, sanitizeFn) {
-  if (!unsafeHtml.length) {
-    return unsafeHtml
+  if (!unsafeHtml || !unsafeHtml.length) {
```

Comment:
How about we just do `!Boolean(unsafeHtml)`?

Reply:
Good point, done the reading and now I knew Boolean(...) is alrady used here in the source base. To be amended

Reply:
So that way:
```js
if (!Boolean(unsafeHtml) || !unsafeHtml.length) {
```
cannot be written, as there is a rule in ESLint config `Redundant Boolean call.eslint(no-extra-boolean-cast)`, which fixes to expression I've used. Not sure what to do with this to not over engineer simple statement

Reply:
I guess we can just do `!unsafeHtml`? @Johann-S what do you prefer?